translations/fr/get5.phrases.txt

```diff
@@ -182,7 +182,7 @@
     }
     "TeamIsReadyForMapSelection"
     {
-        "fr"            "{1} est prêt pour le tour de veto."
+        "fr"            "{1} est prête pour la sélection de map."
```

Comment:
@Iwhite67 This was changed to "prête" over "prêt". Is this intentional? Note that the first word here is a variable; the team's name.